### ISSUE TITLE
Missing required field in OpenAPI JSON body

### ISSUE THREAD

**Issue by Foorack** [getinsomnia/insomnia#3941]
**Describe the bug**

**To Reproduce**
1. Import from URL <https://vrchatapi.github.io/specification/openapi.yaml>
2. Click on 'Request Collection'
3. Scroll down to 'favorites'
4. Show the JSON body.
4. See missing "type" field:

![bild](https://user-images.githubusercontent.com/5008081/130759657-26a0f606-4c3a-458b-b2ac-f64007903b47.png)

**Expected behavior**
Type is clearly marked as a field:
![bild](https://user-images.githubusercontent.com/5008081/130758788-3bca371a-54f9-4965-8447-2ea9dc351fb3.png)
![bild](https://user-images.githubusercontent.com/5008081/130758924-045bb37a-e421-4721-80df-e5f9d658f988.png)

Only difference with "type" compared to others is that it's a $ref field, but $ref fields are working in other places... Could it be that it's a string enum?...
![bild](https://user-images.githubusercontent.com/5008081/130758966-b25e45f0-1766-4920-8cde-bb4d399db00e.png)

**Desktop (please complete the following information):**
 - OS: Windows: 11 (22000.160)
 - Installation Method: EXE from download site
 - App Version: 2021.4.1
![bild](https://user-images.githubusercontent.com/5008081/130759512-3bc80e39-4e4e-4a19-8564-4bbc9558f88a.png)

**Additional context**
No API login is required. Simply open the spec and view the request.

**Comment by thewheat** [comment]
I can't seem to replicate this. @Foorack could you double check to see if this is still any issue?

![image](https://user-images.githubusercontent.com/892961/135744863-67e1e0fe-3ac7-4dfe-84c2-400632628ef4.png)

My test setup details
```
Version: Insomnia 2021.5.3
Release date: 14/09/2021
OS: Darwin x64 20.6.0
Electron: 11.2.3
Node: 12.18.3
V8: 8.7.220.31-electron.0
Architecture: x64
node-libcurl: libcurl/7.73.0 OpenSSL/1.1.1k zlib/1.2.11 brotli/1.0.9 zstd/1.4.9 libidn2/2.1.1 libssh2/1.9.0 nghttp2/1.42.0
```

**Comment by Foorack** [comment]
Hello, Thanks for help test this @thewheat,

This is no longer an issue, as the above link was updated. We ended up simply removing all mentions of read-only.
After thinking about this more, I wouldn't consider this a bug. If you e.g., have a User object with `id` and `name`, and when you send a POST request to /users/{userId} you want to be able to update the name but not the ID, so in this case it makes sense to mark it as read-only, and therefore for it to not show up in Insomnia. Having read-only on this field was an incorrect action done by us.

Closing this as invalid.
